Userland/Libraries/LibJS/Bytecode/Pass/UnifySameBlocks.cpp

```diff
@@ -29,6 +29,8 @@ void UnifySameBlocks::perform(PassPipelineExecutable& executable)
                 continue;
 
             auto candidate_bytes = candidate_block->instruction_stream();
+            // FIXME: NewBigInt holds a SignedBigInteger, which's memory footprint
+            //        does not reflect its value
```

Comment:
this sentence took a good minute to parse
```suggestion
            // FIXME: NewBigInt's value is not correctly reflected by its encoding in memory,
            //        this will yield false negatives for blocks containing that
```
or something of the sort.